eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/TransactionalExecutorProvider.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventsourcing.eventstore;
```

Comment:
Shouldn't this interface reside in axon commons, next to the `TransactionalExecutor` interface?

Reply:
It can't, `ProcessingContext` is not in commons (perhaps it should be, but currently it isn't ;))